docs/data/charts/composition/composition.md

```diff
@@ -218,6 +219,25 @@ To add a grid, you can use the `<ChartsGrid />` component.
 
 See [Axis—Grid](/x/react-charts/axis/#grid) documentation for more information.
 
+### LegendGrid
+
+To add a a legend, you can use the `<ChartsLegend />` component.
+
+:::warning
+The Legend is an HTML element since v8.
+I must be inside the Data Provider to get the data, but outside the Surface since it's not an SVG element.
```

Comment:
```suggestion
It must be rendered inside the `ChartDataProvider` to get the data, but outside the `ChartsSurface` since it's not an SVG element.
```